README.md

```
@@ -1,9 +1,25 @@
-applications.
+This repository houses the policy documents, including Privacy Policies and Contact Support information, for our applications.
+
+## Applications
+
+### AirsoftCalculator
+
+Airsoft Calculator is designed for airsoft players who want to ensure their replica's power does not surpass legal limits.  
+It calculates the energy in joules, allowing users to select the mass of the pellet (in grams) and enter the velocity (in fps or m/s) as measured by their chronograph.
+
+**Supported Languages:**
+- Czech
```

Markdownlint

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)